onedal/common/sycl.cpp

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "onedal/common/sycl_interfaces.hpp"
+#include "onedal/common/pybind11_helpers.hpp"
+
+namespace py = pybind11;
+
+namespace oneapi::dal::python {
+
+#ifdef ONEDAL_DATA_PARALLEL
+
+void instantiate_sycl_interfaces(py::module& m){
+    py::class_<sycl::queue> syclqueue(m, "SyclQueue");
```

Comment:
Please write a comment about the purpose of this class. I.e. that it implements sycl queue interface in case dpctl's sycl queue is not available.

Reply:
Done, let me know what you think

Reply:
I think having those parts (here and pytorch-related) commented is great!